sample-apps/node/frontend-service/index.js

```diff
@@ -77,19 +77,30 @@ app.get('/remote-service', (req, res) => {
   request.end();
 });
 
+// The following logic serves as the async call made by the /client-call API
+let makeAsyncCall = false;
+setInterval(() => {
+  if (makeAsyncCall) {
```

Comment:
maybe no need to check makeAsyncCall. Can we just send the request every 3 seconds at regular interval?

Reply:
That's a good idea, but this is in line with the Java/Python implementation so I think this should be okay for consistency.

Let me know what you think

Reply:
Interesting. How do we know if an async operation is triggered by a sync operation? Especially when the async operation is triggered by `setInterval` which just execute periodically.

Reply:
You're right, they are not connected in the telemetry. Regardless, I think we should follow the implementation of Java/Python for consistency